setup.py

```diff
@@ -32,7 +32,7 @@ def get_git_revision_hash():
 INSTALL_REQUIRES = [
     'Markdown>=2.6,<2.7',
     'bleach>=1.4.2,<2.2',
-    'Wagtail<2.3',
+    'Wagtail>=2.4',
```

Comment:
I think we should just change this to `Wagtail>=1.10,<2.5` for now so we don't make any assumptions about future versions